packages/test/test-end-to-end-tests/src/test/detachedContainerTests.spec.ts

```diff
@@ -811,7 +812,10 @@ describeFullCompat("Detached Container", (getTestObjectProvider) => {
 		// Create a new request url from the resolvedUrl of the first container.
 		assert(container.resolvedUrl);
 		const requestUrl2 = await provider.urlResolver.getAbsoluteUrl(container.resolvedUrl, "");
-		const container2 = await loader.resolve({ url: requestUrl2 });
+		const container2 = await loader.resolve({
+			url: requestUrl2,
+			headers: { [LoaderHeader.cache]: true },
```

Comment:
I assume this change means - we need to open a task to fix the test not to depend on caching, right?